spec/controllers/saml_idp_controller_spec.rb

```diff
@@ -1451,15 +1451,6 @@ def name_id_version(format_urn)
         )
       end
 
-      let(:analytics_hash) do
-        {
-          success: true,
-          request_signed: authn_requests_signed,
-          matching_cert_serial:,
-          encryption_cert_matches_matching_cert:,
-        }
-      end
```

Comment:
i removed this to make the result conditions more explicit in the individual tests